js/views/cfi_navigation_logic.js

```diff
@@ -1299,7 +1321,7 @@ var CfiNavigationLogic = function(options) {
         _.each($elements, function ($node) {
             var node = $node[0];
             var isTextNode = (node.nodeType === Node.TEXT_NODE);
-            var element = isTextNode ? node.parentElement : node;
+            var element = isTextNode ? getParentElement(node) : node;
```

Comment:
There are other uses of `node.parentElement`, notably `plugins/highlights/manager.js` `getAnnotationMidpoints()`
https://github.com/readium/readium-shared-js/blob/develop/plugins/highlights/manager.js#L304
I also note the use of jQuery's `$node.parent()` in many places in ReadiumJS, notably:
```javascript
                //figure out a better way to get the html parent from an element..
                var $html = $element.parent();
```
https://github.com/readium/readium-shared-js/blob/develop/plugins/highlights/manager.js#L322

@JCCR